### PR TITLE
Publish InfluxDB 2.0.7 docs

### PR DESCRIPTION
**Pull request by pierwill**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -8,6 +8,40 @@ menu:
...
+### SQLite Metadata Store
+
+This release adds an embedded SQLite database
+for storing metadata required by the latest UI features like notebooks and annotations.
+
+### Features
+- Add Geo graph type to be able to store in Dashboard cells. <!-- https://github.com/influxdata/influxdb/pull/19811:  -->
+- Add the properties of a static legend for line graphs and band plots. <!-- https://github.com/influxdata/influxdb/pull/21218:  -->
+- Remove feature flags for permanent UI features: <!-- https://github.com/influxdata/influxdb/pull/21531:  -->
+  `axisTicksGenerator`, `legendOrientation`, `mosaicGraphType`, and `bandPlotType`.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:22]: shall we link to these graph types?

File: content/influxdb/v2.0/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -8,6 +8,40 @@ menu:
...
+- Allow hiding the tooltip independently of the static legend. <!-- https://github.com/influxdata/influxdb/pull/21547:  -->
+- Support pagination when listing users via the API. <!-- https://github.com/influxdata/influxdb/pull/21367:  -->
+- Add `influxd` configuration flag `--sqlite-path`
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:25]: see new docs, guessing we'll add link to those

File: content/influxdb/v2.0/reference/release-notes/influxdb.md [hunk 3]
```diff
@@ -8,6 +8,40 @@ menu:
...
+  for specifying a user-defined path to the SQLite database file. <!-- https://github.com/influxdata/influxdb/pull/21543:  -->
+- Update `influxd` configuration flag `--store` to work with string values `disk` or `memory`.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:27]: didn't see this flag in docs, something to add? along with supported values disk/memory/?

File: content/influxdb/v2.0/reference/release-notes/influxdb.md [hunk 4]
```diff
@@ -8,6 +8,40 @@ menu:
...
+This release adds an embedded SQLite database
+for storing metadata required by the latest UI features like notebooks and annotations.
+
+### Features
+- Add Geo graph type to be able to store in Dashboard cells. <!-- https://github.com/influxdata/influxdb/pull/19811:  -->
+- Add the properties of a static legend for line graphs and band plots. <!-- https://github.com/influxdata/influxdb/pull/21218:  -->
+- Remove feature flags for permanent UI features: <!-- https://github.com/influxdata/influxdb/pull/21531:  -->
+  `axisTicksGenerator`, `legendOrientation`, `mosaicGraphType`, and `bandPlotType`.
+- Allow hiding the tooltip independently of the static legend. <!-- https://github.com/influxdata/influxdb/pull/21547:  -->
+- Support pagination when listing users via the API. <!-- https://github.com/influxdata/influxdb/pull/21367:  -->
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:24]: Do we have a task-based list user to add sorting options?
offset, limit, after, descending

File: content/influxdb/cloud/reference/cli/influx/restore/index.md [hunk 5]
```diff
@@ -48,7 +48,6 @@ influx restore [flags]
 | `-h` | `--help`          | Help for the `restore` command                                        |             |                       |
```
[the remark this answers is not in the document]
Comment (pierwill) [reply content/influxdb/cloud/reference/cli/influx/restore/index.md:48]: Done :)

File: content/influxdb/v2.0/reference/release-notes/influxdb.md (hunk 2 above)
Comment (pierwill) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:25]: 👍 - [ ] TODO

File: content/influxdb/v2.0/reference/release-notes/influxdb.md (hunk 3 above)
Comment (pierwill) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:27]: There was some debate about this here: https://github.com/influxdata/docs-v2/pull/2645

Comment (kelseiv) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:27]: makes sense, thank you @pierwill

**Review by kelseiv — APPROVED** [review]

**Pull request merged**